HTML/HTML.sublime-syntax

```diff
@@ -284,6 +318,11 @@ contexts:
         - script-javascript
         - tag-generic-attribute-meta
         - tag-generic-attribute-value
+    - match: (?=(?i:application/json{{unquoted_attribute_break}}|'application/json'|"application/json"))
```

Comment:
https://mimesniff.spec.whatwg.org/#json-mime-type

> A JSON MIME type is any MIME type whose subtype ends in "+json" or whose essence is "application/json" or "text/json".

Reply:
With more complex patterns to be matched a variable makes very much sense.